frontend/src/styles.css

```diff
@@ -6,4 +6,8 @@ html, body{
   height: 100%;
 }
 
+.appContainer{
```

Comment:
Currently I have a div with id `app` in html, and am rendering React components into it.

This class (with a confusing name) is for another div — the main React div (which will be rendered in request of root route, and within which all other React components will be rendered).

I guess, I can just remove the `div id='app'` from index.ejs and render React components directly into the body.